src/commands/config/setClusterServer.ts

```diff
@@ -148,6 +161,10 @@ export default createCommand({
       .option("api-key", {
         desc: "Cluster API key.",
         type: "string",
+      })
+      .option("cluster", {
+        desc: "Cluster type. Options: skygeario, custom.",
```

Comment:
We can use `choices: ["skygeario", "custom"]` instead and map from the available options dynamically.

Reply:
Since we have some hidden options `skygear-staging` and `skygear-dev`, if we set the choices, those options will become unavailable and internal user will need to use `--endpoint` + `--api-key` for using staging cluster. I am open to this, thought?:)

Reply:
We can provide the correct available options using `yargs.argv.debug`.

Reply:
Yes, you are right. Updated, thanks!